css/properties/-webkit-nbsp-mode.json

```
@@ -5,7 +5,8 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "≤15",
```

See: https://github.com/caugner/css-properties/commit/ce826a59d8c6f3c4c56dd1454b4ed7b2398d5773

```suggestion
              "version_added": "1",
```